apps/session-manager/src/services/token.ts

```diff
@@ -8,3 +9,8 @@ import * as crypto from "crypto";
 export const getNewToken = (length: number): string =>
   // Use the crypto.randomBytes as token.
   crypto.randomBytes(length).toString("hex");
+
+export const getNewTokenAsync = (length: number): Promise<string> =>
```

Comment:
is useful have an async task?

Reply:
io-backend has it for parallel execution. do you think we can refactor this part. I only ported the same logic from `io-backend`

Reply:
Put the method as deprecated

Reply:
done in https://github.com/pagopa/io-auth-n-identity-domain/pull/22/commits/efaf8f33c282bf83e278da21b6f4c681efe06acd